pkg/backend/openshiftcluster.go

```diff
@@ -106,6 +106,11 @@ func (ocb *openShiftClusterBackend) handle(ctx context.Context, log *logrus.Entr
 			return ocb.endLease(ctx, log, stop, doc, api.ProvisioningStateFailed, err)
 		}
 		if doc.OpenShiftCluster.Properties.Install == nil {
+			// emit the creation duration
+			duration := time.Now().Sub(doc.CorrelationData.StartTime).Seconds()
+			ocb.m.EmitGauge("backend.openshiftcluster.duration", int64(duration), map[string]string{
```

Comment:
Should this be called something else, or do we remove the other `backend.openshiftcluster.duration` emits?

Reply:
wrong location.  The existing location in pkg/backend/openshiftcluster.go is about right, but it needs to detect that ProvisioningState has changed and only emit in that case.